### DISCUSSION TITLE
Cruiser crashes on Android 13

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#68] [Q&A] [answered]
unfortunately the 3.0.11 verison crashes on Android 13. It is quitted directly after starting the app.

It just says:app closed due to an error and one should wait for a corrected version. It is on a Samsung A52G with 1.1.2023 Google Play version and 1.2.2023 Android Securitypatch, One UI-Version 5.0. Do you need some other information?

Can I somehow restore to previous version on this device only?

**Comment by devemux86** [comment]
@DolomitesTom thanks for the report.

Google Play does not allow app downgrades and would require uninstall anyway.

You can solve it for now by clearing the app data in Android app settings.
**NOTE**: this clears your favorites and recorded tracks!

I see an exception in Google Play console.
I will check it and release an update soon.

**Reply by DolomitesTom** [reply]
Thanks a lot

**Reply by devemux86** [reply]
@DolomitesTom 

Have you successfully restarted 3.0.11 at any time?
Or changed anything in app settings before?

**Reply by DolomitesTom** [reply]
No it directly failed. It previosly installed the patchlevel. It is just a backup phone. Previous version worked. But now the workaround with deleting data has worked.

**Comment by devemux86** [comment]
**_Please everyone always backup (export) your favorites and recorded tracks!_**

**Reply by DolomitesTom** [reply]
Absolutely😁. Good to do if cruiser is installed on 2 phones 🙂. And good at all to have developed an export import function in the past 😏.

**Comment by airald** [comment]
Also crashes to me in a Samsung TAB with Android 13

**Reply by devemux86** [reply]
@airald 

Have you successfully restarted 3.0.11 at any time?
Or changed anything in app settings before?

If you clear app data in Android app settings, does it work?
**NOTE**: this clears your favorites and recorded tracks!

---

There is only one crash report from Honor 8X on Google Play.

**Reply by airald** [reply]
it works again after delete app data

**Reply by airald** [reply]
I have backups of settings and other data that I sync over the cloud to have it available from all my devices. Restore it with Cruiser is just a few taps.

**Reply by devemux86** [reply]
> I have backups of settings

If you restore the app settings from the .pref file, can you reproduce the crash?
If you can reproduce it, can you send me the .pref file to analyze it?

**Reply by airald** [reply]
when I loaded again my backup pref file, all was working properly. The pref file comes from my Xiaomi phone, made yesterday from version 3.0.11. I sent the pref file.

**Reply by devemux86** [reply]
@airald thanks for the pref file!

Did the Xiaomi crash too?

**Reply by airald** [reply]
No, Xiaomi never crashed running Cruiser (and Kurviger 2 in the past).

**Comment by devemux86** [comment]
I released Cruiser 3.0.12 with improvements yesterday.
We are now waiting for Google Play to publish it.

**Comment by devemux86** [comment, marked as answer]
It should be solved in [Cruiser 3.0.12](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

Please let me know if you find anything else.

**Reply by airald** [reply]
No issues when updated my Samsung Tab (Android 13)

**Comment by DolomitesTom** [comment]
Update also works now under Android 13. No new crash. But what I have just recognized is that the switch for turning instructions is not working any more (also in version 3.0.11). Even if they are switched off, they are shown in yellow... by this way an addtional question:the yellow turning instructions contain the distances to and from the next wp and from the start to the end. Why is this not shown any more on shaping points? Can this also be activated for shaping points, so that one can see the distances to start and end point?

**Reply by 0709wiwiwi** [reply]
https://forum.kurviger.de/t/zeit-km-zu-shaping-point/9198/5?u=0709
(that was the decision of the community)

**Reply by devemux86** [reply]
There are 3 different zoom options in "Settings | Routing | Appearance":
- ![df_gen_031b](https://user-images.githubusercontent.com/3484020/223723609-697e5bbc-fb9a-49ed-80c8-8ef49a6ebddc.png) Via points (with numbers): zoom 2 - 20
- ![df_gen_031c](https://user-images.githubusercontent.com/3484020/223723680-7abd684c-b00f-4db6-98a2-36fab84d0def.png) Shaping points (with numbers): zoom 2 - 20
- ![df_gen_032c](https://user-images.githubusercontent.com/3484020/223723732-e955d6ed-e891-4e50-acdf-133b1ad31541.png) Turn instructions: Off, zoom 8 - 20

I tested them again and their zoom settings seem to work?

---

> the yellow turning instructions contain the distances to and from the next wp and from the start to the end. Why is this not shown any more on shaping points?

- Via points show distance / time to previous / next via point and start / destination
(info bubbles and waypoints list)
- Shaping points are for shaping the route and in order to not overload the screens,
they do not show any extra information
- Turn instructions show distance / time to next turn instruction
(info bubbles and turn instructions list)

**Comment by DolomitesTom** [comment]
The turn instruction zoom is working fine. There is an editional setting for turn instruction switch on or off. This is not working, off is not switching it off. But i have changed now the zoom of turning instructions to thirteen and this is working fine for me. Thanks

**Reply by devemux86** [reply]
Are we talking about the marked option in the image?

Testing here, when it is off, the turn instruction yellow circles do not appear.
If you see something else, do you follow some steps and can reproduce it?

<img src="https://user-images.githubusercontent.com/3484020/223785829-de552f83-840d-47ca-af71-353760792b8c.jpg" width="50%"/>

**Comment by DolomitesTom** [comment]
I mean Settings -- Navigation -- Display -- turning instructions
![Screenshot_20230308-190942_Cruiser](https://user-images.githubusercontent.com/121371629/223796043-7b8cbe98-63f5-45be-a078-a517d1ba7af1.jpg)

**Reply by devemux86** [reply]
The settings on this screen are to enable / disable the various navigation panels.

The turn instruction is the top horizontal panel (with the small direction symbol),
that displays the next turn instruction and the street name of the turn.

<img src="https://user-images.githubusercontent.com/3484020/213869112-58f27d0f-b0b7-4ac7-b729-382877d6ba98.jpg" width="50%"/>

**Reply by devemux86** [reply]
If you thought it was about the turn instruction yellow circles on the map,
maybe we need a better name for this setting? :slightly_smiling_face:

**Reply by DolomitesTom** [reply]
obviously yes or I am just too old and forgot the usage since I deactivated this years ago 😁

better maybe to write, also for the other settings which refer to the nav. bars: show in navigation panel, 
in german:
"in der Informationsleiste" OR
"im Informationspanel" OR
"in der Navigationsleiste"

to my opinion "Informationspanel" (2nd option) should any German understand.
